Controls/Mail/Mailman.c

```diff
@@ -11,8 +13,21 @@
  * Once the letter is deposited, signal the semaphore (effectively putting up the flag).
  * Print "Finished!" at the end.
  */
+//extern char* mailbox;
 void depositLetter(void) {
 	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+	//int i;
+	//char max=random()%255;//different lengths at diff times
+	for(int i=0;i<255;i++){
+		mailbox[i]=64+rand()%63;
+	}
+	mailbox[256]=0;//null at end
```

Comment:
The last character of mailbox won't be index 256, but actually 255 (the size of mailbox is 256, which is indices 0-255). Again, a macro define for the mailbox size in _Mail.h_ would be nice as then we can change the size of the mailbox if we need to later (plus we don't have this random number 256 here).